egs/multi_en/s5/conf/ami_ihm/mfcc.conf

```diff
@@ -0,0 +1,4 @@
+--use-energy=false
+--sample-frequency=16000
```

Comment:
I think it  might make people's lives easier later on-- so yes.



On Mon, Jun 27, 2016 at 12:49 PM, Allen Guo <notifications@github.com>
wrote:

> In egs/multi_en/s5/conf/ami_ihm/mfcc.conf
> <https://github.com/kaldi-asr/kaldi/pull/771#discussion_r68643294>:
>
> > @@ -0,0 +1,4 @@
> > +--use-energy=false
> > +--sample-frequency=16000
>
> @danpovey <https://github.com/danpovey>: So you'd recommend switching to
> downsampling and retraining the HMM-GMM models from scratch before starting
> TDNN training?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/771/files/d13323180b98a83622b0cb193a3a5cd042468609#r68643294>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADJVuwRCr5al9xNMdcfH9g0uisGZ7rmBks5qQCkygaJpZM4IZ3x7>
> .
>